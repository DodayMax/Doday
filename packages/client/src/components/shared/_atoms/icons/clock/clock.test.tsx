import * as React from 'react';
import { shallow } from 'enzyme';
import { Icons } from '@shared';

describe('Clock', () => {
  it('renders correctly without props', () => {
    const clock = shallow(<Icons.Clock />);

    expect(clock).toMatchSnapshot();
  });
});
