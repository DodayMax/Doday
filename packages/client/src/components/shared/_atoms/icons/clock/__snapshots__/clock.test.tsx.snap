// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Clock renders correctly without props 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <g
        fill="black"
        fillRule="evenodd"
      >
        <path
          d="M12.108 1.196c-5.94 0-10.756 4.815-10.756 10.755s4.815 10.756 10.756 10.756c5.94 0 10.755-4.816 10.755-10.756S18.048 1.196 12.108 1.196zm0 20.044a9.289 9.289 0 1 1 0-18.578 9.289 9.289 0 0 1 0 18.578z"
        />
        <path
          d="M16.996 11.462h-4.4V6.084a.733.733 0 1 0-1.466 0v6.845h5.866a.733.733 0 1 0 0-1.467z"
        />
      </g>,
      "height": 20,
      "viewBox": "0 0 24 24",
      "width": 20,
      "xmlns": "http://www.w3.org/2000/svg",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <path
            d="M12.108 1.196c-5.94 0-10.756 4.815-10.756 10.755s4.815 10.756 10.756 10.756c5.94 0 10.755-4.816 10.755-10.756S18.048 1.196 12.108 1.196zm0 20.044a9.289 9.289 0 1 1 0-18.578 9.289 9.289 0 0 1 0 18.578z"
          />,
          <path
            d="M16.996 11.462h-4.4V6.084a.733.733 0 1 0-1.466 0v6.845h5.866a.733.733 0 1 0 0-1.467z"
          />,
        ],
        "fill": "black",
        "fillRule": "evenodd",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M12.108 1.196c-5.94 0-10.756 4.815-10.756 10.755s4.815 10.756 10.756 10.756c5.94 0 10.755-4.816 10.755-10.756S18.048 1.196 12.108 1.196zm0 20.044a9.289 9.289 0 1 1 0-18.578 9.289 9.289 0 0 1 0 18.578z",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M16.996 11.462h-4.4V6.084a.733.733 0 1 0-1.466 0v6.845h5.866a.733.733 0 1 0 0-1.467z",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
      ],
      "type": "g",
    },
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <g
          fill="black"
          fillRule="evenodd"
        >
          <path
            d="M12.108 1.196c-5.94 0-10.756 4.815-10.756 10.755s4.815 10.756 10.756 10.756c5.94 0 10.755-4.816 10.755-10.756S18.048 1.196 12.108 1.196zm0 20.044a9.289 9.289 0 1 1 0-18.578 9.289 9.289 0 0 1 0 18.578z"
          />
          <path
            d="M16.996 11.462h-4.4V6.084a.733.733 0 1 0-1.466 0v6.845h5.866a.733.733 0 1 0 0-1.467z"
          />
        </g>,
        "height": 20,
        "viewBox": "0 0 24 24",
        "width": 20,
        "xmlns": "http://www.w3.org/2000/svg",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <path
              d="M12.108 1.196c-5.94 0-10.756 4.815-10.756 10.755s4.815 10.756 10.756 10.756c5.94 0 10.755-4.816 10.755-10.756S18.048 1.196 12.108 1.196zm0 20.044a9.289 9.289 0 1 1 0-18.578 9.289 9.289 0 0 1 0 18.578z"
            />,
            <path
              d="M16.996 11.462h-4.4V6.084a.733.733 0 1 0-1.466 0v6.845h5.866a.733.733 0 1 0 0-1.467z"
            />,
          ],
          "fill": "black",
          "fillRule": "evenodd",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M12.108 1.196c-5.94 0-10.756 4.815-10.756 10.755s4.815 10.756 10.756 10.756c5.94 0 10.755-4.816 10.755-10.756S18.048 1.196 12.108 1.196zm0 20.044a9.289 9.289 0 1 1 0-18.578 9.289 9.289 0 0 1 0 18.578z",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M16.996 11.462h-4.4V6.084a.733.733 0 1 0-1.466 0v6.845h5.866a.733.733 0 1 0 0-1.467z",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
        ],
        "type": "g",
      },
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
